osu.Game/Screens/Edit/BindableBeatDivisor.cs

```diff
@@ -29,10 +29,11 @@ public BindableBeatDivisor(int value = 1)
         /// Set a divisor, updating the valid divisor range appropriately.
         /// </summary>
         /// <param name="divisor">The intended divisor.</param>
-        public void SetArbitraryDivisor(int divisor)
+        /// <param name="force">Ignores the current valid divisor range when true.</param>
```

Comment:
This reads wrong. Don't you mean "forces changing the valid divisors to a common preset"? I'd probably name this variable `preferKnownPresets` or experiment with extracting the logic out to a second method for this, called something like `SetPresetsForDivisor(int divisor)`.